### PR TITLE
[14.0][IMP] *_import_online: always update next_run to cover current period

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

**Review by alexey-pelykh — APPROVED** [review]
Looks good, optimization! Just adjust the commit message to be more specific

**Comment by NL66278** [comment]
@alexey-pelykh I changed the commit message. Thanks for your review!

File: account_statement_import_online/models/online_bank_statement_provider.py [hunk 1]
```diff
@@ -69,7 +69,10 @@ class OnlineBankStatementProvider(models.Model):
         string="Update Schedule",
         compute="_compute_update_schedule",
     )
-    last_successful_run = fields.Datetime(string="Last successful pull")
+    last_successful_run = fields.Datetime(
+        string="Last successful pull",
+        readonly=True,
```
Comment (pedrobaeza) [inline account_statement_import_online/models/online_bank_statement_provider.py:74]: Why doing this readonly? What if I want to change it manually?

**Review by pedrobaeza — COMMENTED** [review]
I have problems understanding what this is fixing.

File: account_statement_import_online/models/online_bank_statement_provider.py (hunk 1 above)
Comment (NL66278) [reply account_statement_import_online/models/online_bank_statement_provider.py:74]: @pedrobaeza Why would you change this manually? Is is literally the last time the job last run.

Comment (pedrobaeza) [reply account_statement_import_online/models/online_bank_statement_provider.py:74]: OK, if it's not used for any sync related thing.

Comment (NL66278) [reply account_statement_import_online/models/online_bank_statement_provider.py:74]: @pedrobaeza As for what it is fixing: suppose the server running Odoo is not always active (for upgrade or other maintenance etc.). It means that the next_run which is supposed to be updated at regular intervals (example every 4 hours), is not always updated and will get further and further back in time. As the `_scheduled_pull` method only retrieves data until the next_run date, you will be getting data back with ever greater delays.  This makes sure that at the end of each run, when the next_run is increased by the run interval, the next_run will be in the future.

Comment (pedrobaeza) [reply account_statement_import_online/models/online_bank_statement_provider.py:74]: And why not simply change the fetch to get from last_run to now?

Comment (NL66278) [reply account_statement_import_online/models/online_bank_statement_provider.py:74]: Simply running until now would also solve the problem I outlined. Adjusting the next_run has the same effect with the added advantage of correctly showing when the next run will be.

Comment (pedrobaeza) [reply account_statement_import_online/models/online_bank_statement_provider.py:74]: Well, I always prefer the simplest formula, but as you prefer.

**Pull request merged**